### ISSUE TITLE
Extend addons filtering with addon "disabled" and "enabled" options

### ISSUE THREAD

**Issue by glogiotatidis** [mozmeao/snippets-service#504]
Extend issue #343 to include addon enabled and "disabled" options. This is currently blocked by 

https://bugzilla.mozilla.org/show_bug.cgi?id=1462124

**Comment by glogiotatidis** [comment]
Info will not be available before Fx62

**Comment by glogiotatidis** [comment]
Tried getting the info with 

```
gSnippetsMap.getAddonsInfo().then(info => console.log(info))
```

and while I get a list of addons, now the disabled addons are not listed at all. The expected outcome would be that the addon is listed the `userDisabled` is set to `true`. 

We originally discussed this in https://github.com/mozmeao/snippets-service/issues/343#issuecomment-387399040

@k88hudson do you have any insight on this?

**Comment by glogiotatidis** [comment]
Latest update is that Kate will talk about this with the Addon folks

**Comment by glogiotatidis** [comment]
Re-opening b/c this information does not exist in ASR targeting :|

I commented in the bug to try and get this fixed.

Meanwhile we'll implement only "installed" and "not installed" functionality for ASR to be in par with what we have in current system

**Comment by glogiotatidis** [comment]
Re-opening b/c this information does not exist in ASR targeting :|

Created new bug [149432](https://bugzilla.mozilla.org/show_bug.cgi?id=1494232)

Meanwhile we'll implement only "installed" and "not installed" functionality for ASR to be in par with what we have in current system

**Comment by glogiotatidis** [comment]
Moving this to Fx64 based on discussion with the ASR team.

**Comment by glogiotatidis** [comment]
[Update in the bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1494232#c3) clarifies that this won't be in Fx64 unless uplifted. Will be in Fx65.

**Comment by glogiotatidis** [comment]
New PR to fix this in AS

**Comment by glogiotatidis** [comment]
According to the bug this is planned for Fx68 scheduled for 2019-07-09 .

/remind me to check if this has been fixed in nightly in one month

**Comment by glogiotatidis** [comment]
The PR https://github.com/mozilla/activity-stream/pull/4757 is still unmerged. 

/remind me to check if this has been fixed in nightly in two months

**Comment by glogiotatidis** [comment]
Nope, still not landed.

@mozmeao/omn-team is this important? Should we bring it to AS team's attention?

/remind me to check in 2 months

**Comment by jkt230** [comment]
Yes we should. This is going to become important as we start to target those who are "Caring Confidentials" and "Feature Forward" per Venetia and Cherry's study of our users. Specific Add-ons is one way we can target them. I am still waiting on a specific list from them.

**Comment by glogiotatidis** [comment]
Per last mtg we still want to have this but not high priority

**Comment by glogiotatidis** [comment]
Bug has been de-priotised. let's talk about wanted targeting attributes in 2019H2
